controllers/storageautoscaler/util.go

```diff
@@ -0,0 +1,109 @@
+package storageautoscaler
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/go-logr/logr"
+	prometheusapi "github.com/prometheus/client_golang/api"
+	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	prometheusconfig "github.com/prometheus/common/config"
+	"github.com/prometheus/common/model"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+	"github.com/red-hat-storage/ocs-operator/v4/controllers/platform"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func getPrometheusURL(operatorNamespace string) (string, error) {
+	prometheusURL := "https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091"
```

Comment:
I wonder if this url and/or port is configurable by the cluster admin? If so, our query could fail.

Reply:
No they are not configurable as re-searched, prometheus is deployed on `openshift-monitoring` namespace and for rosa it is different